.github/workflows/test_conda_store_server_integration.yaml

```diff
@@ -0,0 +1,82 @@
+name: "Test conda-store-server (integration)"
+
+env:
+  DEFAULT_PYTHON_VERSION: "3.10"
+  FORCE_COLOR: "1" # Make tools pretty.
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+
+# ensuring only one instance is running at a given time
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  integration-test-conda-store-server:
+    name: "integration-test conda-store-server"
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        test-type: ["playwright", "integration", "user-journey"]
+    defaults:
+      run:
+        shell: bash -el {0}
+        working-directory: conda-store-server
+    steps:
+      - name: "Checkout Repository 🛎"
+        uses: actions/checkout@v4
+
+      - name: "Set up env 🐍"
+        uses: conda-incubator/setup-miniconda@v3
+        with:
+          environment-file: conda-store-server/environment-dev.yaml
+          miniforge-version: latest
+
+      - name: "Install build dependencies 📦"
+        run: |
+          pip install hatch
+          sudo apt install wait-for-it -y
+
+      - name: "Deploy docker-compose"
+        run: |
+          docker-compose up -d
+          docker ps
+
+          wait-for-it localhost:5432 # postgresql
+          wait-for-it localhost:9000 # minio
+          wait-for-it localhost:8080 # conda-store-server
+          sleep 10 # give the server more time to initialize
```

Comment:
This gives time for envs that come with the server to initialize (default, filesystem).

Reply:
`wait-for-it` just checks whether a URL is accessible. But some additional work is needed on the server side to initialize the bundled envs (default, filesystem), which are used for testing. So `wait-for-it` alone is not sufficient.